docs/_layouts/global.html

```diff
@@ -14,11 +14,165 @@
+        /*card borders*/
+        #current-features + ul li:nth-child(1) {
+          border-color: rgb(46, 180, 77);
+          background-color: rgba(46, 180, 77, 0.1);
+        }
+        #current-features + ul li:nth-child(2) {
+          border-color: rgb(242, 144, 58);
+          background-color: rgba(242, 144, 58, 0.1);
+        }
+        #current-features + ul li:nth-child(3) {
+          border-color: rgb(36, 140, 203);
+          background-color: rgba(36, 140, 203, 0.1);
+        }
+        #current-features + ul li:nth-child(4) {
+          border-color: rgb(233, 106, 121);
+          background-color: rgba(233, 106, 121, 0.1);
+        }
+        #current-features + ul li:nth-child(5) {
+          border-color: rgb(20, 103, 103);
+          background-color: rgba(20, 103, 103, 0.1);
+        }
+        #current-features + ul li:nth-child(6) {
+          border-color: rgb(250, 188, 66);
+          background-color: rgba(250, 188, 66, 0.1);
+        }
+        /*card titles*/
+        #current-features + ul li:nth-child(1) strong {
+          background-color: rgb(46, 180, 77);
+        }
+        #current-features + ul li:nth-child(2) strong {
+          background-color: rgb(242, 144, 58);
+        }
+        #current-features + ul li:nth-child(3) strong {
+          background-color: rgb(36, 140, 203);
+        }
+        #current-features + ul li:nth-child(4) strong {
+          background-color: rgb(233, 106, 121);
+        }
+        #current-features + ul li:nth-child(5) strong {
+          background-color: rgb(20, 103, 103);
+        }
+        #current-features + ul li:nth-child(6) strong {
+          background-color: rgb(250, 188, 66);
+        }
+        /*custom font sizes to make content fit nicely*/
+        #current-features + ul li:nth-child(1) p {
+          font-size: 16px;
+        }
+        #current-features + ul li:nth-child(3) strong {
+          font-size: 14px;
+        }
+        #current-features + ul li:nth-child(4) p {
+          font-size: 16px;
+        }
+        #current-features + ul li:nth-child(5) p {
+          font-size: 18px;
+        }
+        #current-features + ul li:nth-child(6) p {
+          font-size: 20px;
+        }
+        #current-features + ul li p a {
+          text-decoration: underline;
+          color: inherit;
+        }
+        #current-features + ul li p {
+          text-align: left;
+          font-size: 12px;
+          line-height: 1.6;
+          padding-top: 30px;
+        }
+        #current-features + ul li p strong {
+          display: block;
+          font-size: 16px;
+          width: 100%;
+          position: absolute;
+          left: 0;
+          top: 0;
+          text-align: center;
+          color: white;
+          padding: 5px 0;
+        }
+        #current-features + ul li p code {
+          background-color: #F7F4F3;
+          border: 1px solid #C4CBCA;
+          font-size: 100%;
+          padding: 0 2px;
+        }
     </style>
```

Comment:
This doesn't have to be in this PR, but is there a simple way to pull out all of this style into a separate css file?